src/components/TimelineEvent.js

```diff
@@ -1,10 +1,25 @@
 import React from 'react';
 import './TimelineEvent.css';
 import Timestamp from './Timestamp';
+import PropTypes from 'prop-types';
 
-const TimelineEvent = () => {
+const TimelineEvent = (props) => {
+  return(
+  <div>
+    <ul>
+      <li className="event-person">{props.person}</li>
+      <li className="event-status">{props.status}</li>
+      <li className="event-time">{props.timeStamp}</li>
```

Comment:
You can use the timestamp component to show the time.  This is why that one test isn't passing. 

```suggestion
      <li className="event-time"><Timestamp time={props.timeStamp} /></li>
```